linodecli/plugins/region-table.py

```diff
@@ -0,0 +1,53 @@
+"""
+The region-table plugin displays a table output
+for the capabilities of each region.
+"""
+import sys
+
+from prettytable import PrettyTable
+
+
+def call(_, ctx):
+    """
+    Invokes the region-table plugin
+    """
+    status, regions = ctx.client.call_operation("regions", "list")
+    capabilities_short = [
+        "Linodes",
+        "GPU",
+        "Premium",
+        "NB",
+        "Blocks",
+        "Objects",
+        "K8s",
+        "FW",
+        "Vlans",
+        "DB",
+    ]
+    capabilities = [
+        "Linodes",
+        "GPU Linodes",
+        "Premium Plans",
+        "NodeBalancers",
+        "Block Storage",
+        "Object Storage",
+        "Kubernetes",
+        "Cloud Firewall",
+        "Vlans",
+        "Managed Databases",
+    ]
```

Comment:
optional: Maybe we could use tuples to couple the short-form and long-form capability names
```python
capabilities = [
    ("Linodes", "Linodes"),
    ("GPU Linodes", "GPU"),
    ("Premium Plans", "Premium"),
    ("NodeBalancers", "NB"),
    ("Block Stoarge", "Blocks"),
    ("Object Storage", "Objects"),
    ("Kubernetes", "K8s"),
    ("Firewalls", "FW"),
    ("Vlans", "Vlans"),
    ("Managed Databases", "DB"),
]
```

Reply:
@lgarber-akamai 
Could you also add Metadata Service to the column?

Reply:
cc @jriddle-linode